subsys/net/lib/lwm2m_client_utils/lwm2m/include/location_assistance.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef LWM2M_CLIENT_UTILS_LOCATION_ASSISTANCE_INTERNAL_H__
+#define LWM2M_CLIENT_UTILS_LOCATION_ASSISTANCE_INTERNAL_H__
+
+#include <zephyr/kernel.h>
+
+#define LOCATION_ASSISTANT_RESULT_TIMEOUT 180
+
+#define LOCATION_ASSISTANT_UPDATE_PERIOD 60
+#define LOCATION_ASSISTANT_INITIAL_RETRY_INTERVAL 675
+#define LOCATION_ASSISTANT_MAXIMUM_RETRY_INTERVAL 86400
+
+struct assistance_retry_data {
+	struct k_work_delayable worker;
+	int timeout_period;
+	int retry_left;
+	int retry_seconds;
+	bool temp_error;
+};
+
+/**
+ * @brief Initialize the location assistance library and without re-send.
+ *
+ * @return Returns a negative error code (errno.h) indicating
+ *         reason of failure or 0 for success.
+ */
+int location_assistance_disable_resend(void);
```

Comment:
If this only does ` location_assistance_retry_init(false);` and always returns zero, and also this is an internal API, 
Why don't we move the `location_assistance_retry_init()` into here?

At least the documentation is wrong. Why is this even called `disable_something` when the documentation is `initialize something`
Why error code are documented if the always returns zero. Why return is not `void` if this cannot fail?

Reply:
I replace old funktion by new one: location_assistance_retry_init() and update documentation